src/style.css

```diff
@@ -1,3 +1,71 @@
+html {
+  font-family: Roboto, Arial, sans-serif;
+
+  --link-main-color: #000;
+  --link-active-color: #00acdc;
+}
+
 body {
+  margin: 0 auto;
+}
+
+.header {
+  padding: 0 50px;
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+}
+
+.header_logo {
+  width: 40px;
+  height: 40px;
+  display: flex;
+  justify-content: center;
+}
+
+.nav__list {
   margin: 0;
+  padding: 0;
+  list-style: none;
+  display: flex;
+  justify-content: space-between;
+}
+
+.nav__item {
+  margin: 0 0 0 20px;
```

Comment:
Be consistent with your vertical margins (Add only top or only bottom margin, but not both). In this case, it seems that you are only adding a left margin, which is good for horizontal consistency, but make sure to review the vertical margins throughout the project to ensure they are consistent as well.